bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/Auditor.java

```diff
@@ -1230,7 +1232,16 @@ void checkAllLedgers() throws BKException, IOException, InterruptedException, Ke
                     return;
                 }
 
+                try {
+                    semaphore.acquire();
+                } catch (InterruptedException e) {
+                    LOG.error("Unable to acquire open ledger operation semaphore ", e);
```

Comment:
add `Thread.currentThread().interrupt();`

Reply:
@hangc0276 
`Thread.currentThread().interrupt()` is to be called only in case of `InterruptedException`
we need to separate the two catch clauses

Reply:
@eolivelli Ok, i just return when tryAcquire failed.